distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-fanout-framework-contents.adoc

```diff
@@ -0,0 +1,270 @@
+
+==== Catalog Fanout Framework 
+
+The Fanout Catalog Framework (`fanout-catalogframework` bundle) provides an implementation of the Catalog Framework that acts as a proxy, federating requests to all available sources. 
```

Comment:
I've made some edits, but I suspect there's still some rewriting to be done here.